media/doc/README.WINE

```diff
@@ -286,6 +286,9 @@ kernel32 -
   reactos/dll/win32/kernel32/winnls/string/nls.c         # Synced in r52754
   reactos/dll/win32/kernel32/winnls/string/sortkey.c     # Synced to WineStaging-3.3
 
+lpk
+  reactos/dll/win32/lpk/bidi.c    #Synced to WineStaging-3.7
```

Comment:
Can you add some extra (space) indenting for the "# Synced to ..." text? (as is done in the rest of the file).